Assets/GothicVR/Scripts/Caches/AssetCache.cs

```diff
@@ -196,7 +196,15 @@ public static IMorphMesh TryGetMmb(string key)
             if (MmbCache.TryGetValue(preparedKey, out var data))
                 return data;
 
-            var newData = new MorphMesh(GameData.Vfs, $"{preparedKey}.mmb").Cache();
+            IMorphMesh newData = null;
+            try
```

Comment:
Oh. Don't tell me there are object again, where the mmb file doesn't exist? :-)

Reply:
just in case 😉